### ISSUE TITLE
Persisting LoginState for ArcanaWallet

### ISSUE THREAD

**Issue by synapsecode** [shaloo/sample-auth-flutter#1]
Hey!
Was using this code as reference but noticed one thing. If i close the application and open it again, I have to go through the google sign-in process again.
It is really inconvenient for the user. Any clues as to how I can implement a silent login / persist the login state so that the experience for the end user is smooth?

I believe this is a very important issue, Thanks.

**Comment by shaloo** [comment]
Yes, apps can use `canReconnect` and `reconnect` methods of the `AuthProvider` in the upcoming v1.0.6 and make sure user does not have to re-login if they refresh the page or opens a new tab within 30-min login window. 

Hope that will address your requirement.

PS:  You may also want to check out SendIt app that uses Arcana Auth and allows sending crypto using email/social login. See https://sendit.arcana.network/

**Comment by synapsecode** [comment]
Thanks for the update! Appreciate it. Any estimated date for the release of 1.0.6?

**Comment by shaloo** [comment]
Its under testing but you can take a look at `npm.js` :-)
